pkg/remoteconfig/templates/avalanche-node.tmpl

```diff
@@ -2,6 +2,7 @@
 	"http-host": "{{.HTTPHost}}",
 	"api-admin-enabled": {{.APIAdminEnabled}},
 	"index-enabled": {{.IndexEnabled}},
+	"proposervm-use-current-height-bool": {{.ProposerVMUseCurrentHeight}},
```

Comment:
just set it to `true` here, no need to pass variables for something that is always `true`